client/state/selectors/eligible-for-free-to-paid-upsell.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
```

Comment:
You probably don't need the External dependencies section here since there aren't any.